### PR TITLE
Update 'deployment update' command

### PR DESCRIPTION
**Pull request by pigbug419**
This PR update `pf deployment update` command for update scaler.

### CODE REVIEW

File: pfcli/deployment.py [hunk 1]
```diff
@@ -441,22 +442,17 @@ def create(
...
 def update(
     deployment_id: str = typer.Argument(..., help="Deployment id to update."),
-    replicas: int = typer.Option(
-        ..., "--replicas", "-r", help="Number of replicas to scale deployment."
+    scaler: str = typer.Argument(
+        ..., callback=ast.literal_eval, help="Scaler for deployment."
     ),
 ):
     """Update deployment.
     # TODO: Add more update options.
```
Comment (kdh0102) [inline pfcli/deployment.py:450]: This line is printed when running `pf deployment update --help`.
```suggestion
```

File: pfcli/deployment.py [hunk 2]
```diff
@@ -441,22 +442,17 @@ def create(
 @app.command()
 def update(
     deployment_id: str = typer.Argument(..., help="Deployment id to update."),
-    replicas: int = typer.Option(
-        ..., "--replicas", "-r", help="Number of replicas to scale deployment."
+    scaler: str = typer.Argument(
+        ..., callback=ast.literal_eval, help="Scaler for deployment."
     ),
 ):
     """Update deployment.
```
Comment (kdh0102) [inline pfcli/deployment.py:449]:
```suggestion
    """[Experimental] Update deployment.
```

**Review by kdh0102 — CHANGES_REQUESTED** [review]
Minor comments on descriptions.

**Comment by pigbug419** [comment]
@kdh0102 Thank you for your comments. I've address them!

**Review by kdh0102 — APPROVED** [review]
LGTM!

**Pull request merged**